src/features/vsUtils.ts

```diff
@@ -115,7 +115,15 @@ export const toDiagnostic = (
   );
 
   diagnostic.source = backend;
-  diagnostic.code = alert.Check;
+
+  if (alert.Link.length > 0) {
+    diagnostic.code = {
+      value: alert.Check,
+      target: vscode.Uri.parse(alert.Link)
```

Comment:
@jdkato I can't quite tell from this, what does this url lead to?

Reply:
Whatever the rule's `link:` key defines. Typically this is a page of the parent style guide (such MS, Google, IBM, etc.) that provides some context for the alert.